omnigibson/robots/tiago.py

```diff
@@ -229,31 +224,31 @@ def untucked_default_joint_pos(self):
         pos[self.base_idx] = self.get_joint_positions()[self.base_idx]
         pos[self.trunk_control_idx] = 0.02 + self.default_trunk_offset
         pos[self.camera_control_idx] = np.array([0.0, 0.45])
-        pos[self.gripper_control_idx[self.default_arm]] = np.array([0.045, 0.045])  # open gripper
-
-        # Choose arm based on setting
-        if self.default_arm_pose == "vertical":
-            pos[self.arm_control_idx[self.default_arm]] = np.array(
-                [0.22, 0.48, 1.52, 1.76, 0.04, -0.49, 0]
-            )
-        elif self.default_arm_pose == "diagonal15":
-            pos[self.arm_control_idx[self.default_arm]] = np.array(
-                [0.22, 0.48, 1.52, 1.76, 0.04, -0.49, 0]
-            )
-        elif self.default_arm_pose == "diagonal30":
-            pos[self.arm_control_idx[self.default_arm]] = np.array(
-                [0.22, 0.48, 1.52, 1.76, 0.04, -0.49, 0]
-            )
-        elif self.default_arm_pose == "diagonal45":
-            pos[self.arm_control_idx[self.default_arm]] = np.array(
-                [0.22, 0.48, 1.52, 1.76, 0.04, -0.49, 0]
-            )
-        elif self.default_arm_pose == "horizontal":
-            pos[self.arm_control_idx[self.default_arm]] = np.array(
-                [0.22, 0.48, 1.52, 1.76, 0.04, -0.49, 0]
-            )
-        else:
-            raise ValueError("Unknown default arm pose: {}".format(self.default_arm_pose))
+        # Choose arm joint pos based on setting
+        for arm in self.arm_names:
+            pos[self.gripper_control_idx[arm]] = np.array([0.045, 0.045])  # open gripper
```

Comment:
nice!! so this works for both arms? (i.e.: the joints for L / R are mirrored so that the same joint values result in the mirrored pose between the two?)

Reply:
Yes! works for both arms